docs/src/components/typography/monospace-font.module.css

```diff
@@ -13,13 +16,14 @@
   text-align: center;
 }
 .fontLetter {
+  composes: glyphs;
```

Comment:
🍾

Reply:
Just thought I'd introduce another class composition pattern into this codebase. :)

Lol, I'm not sure what your emoji means.